machine_management/adding-rhel-compute.adoc

```diff
@@ -11,6 +11,13 @@ include::modules/rhel-compute-overview.adoc[leveloffset=+1]
 
 include::modules/rhel-compute-requirements.adoc[leveloffset=+1]
 
+include::modules/rhel-images-aws.adoc[leveloffset=+1]
+
+[NOTE]
+====
+When creating a {op-system-base} compute machine for AWS, ensure that the AMI is {op-system-base} 7. To do so, use the following command option: `--filters "Name=name,Values=RHEL-7.?*GA*"`.
```

Comment:
This `RHEL-7.?*GA*` filter renders with the `GA` part bolded and missing the important asterisks.

Reply:
Great catch! Per Kristine's suggestion, since this is limited to just RHEL workers and the version should be 7.9, I changed all references from RHEL-8 to `Values=RHEL-7.9*`. This should render correctly.